src/Check/MysqlCheck.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace PrestaShop\Module\HealthCheck\Check;
+
+use Laminas\Diagnostics\Check\PDOCheck;
+
+class MysqlCheck extends PDOCheck
+{
+    public function __construct(string $host, string $dbName, string $username, string $password, $port = 3306, int $timeout = 1)
+    {
+        $dsn = sprintf('mysql:host=%s;port=%d;dbname=%s', $host, intval($port), $dbName);
+
+        parent::__construct($dsn, $username, $password, $timeout);
```

Comment:
I think you can't execute this without being already connected to the database, PrestaShop can't be ran without a database access :thinking: 
Dit you try to stop your Sql instance and run the healthcheck?

Reply:
you right. I get an error 500
Maybe I need to perform a query to be sure DB is available. 

The best check would be to check all required tables exists and we can do CRUD on it but it's a huge work and hard to maintain

Reply:
done

Reply:
I think you can remove this check and all things around the database right now because PrestaShop can't work without tables, admin access, or everything around the database.

Reply:
It was the only one that makes sense to me :'(
If you have any idea of what check we can add by default, I will take it ;)

Reply:
removed

Reply:
To add this kind of healthcheck, it must be a separated file from the Core and do not use anything for it.